File: unit11-hw/ruby/birdtest.rb
# birdtest.rb | Tests out the menagerie of bird classes.
#
# Begun by: Dr. Jump for CS 315 at King's College.
# Completed by: Robert Lowman
# Date: 11/29/16
######################################################

require_relative 'Duck.rb'
require_relative 'Goose.rb'
require_relative 'Owl.rb'
require_relative 'Ostrich.rb'
require_relative 'Kiwi.rb'
require_relative 'Penguin.rb'

bird0 = Bird.new "Hawkeye "
bird0.print

bird1 = Duck.new "Donald "
bird1.print

bird2 = Goose.new "Mother "
bird2.print

bird3 = Owl.new "Woodsey "
bird3.print

bird4 = Ostrich.new "Orville "
bird4.print

bird5 = Penguin.new "Petey "
bird5.print

bird6 = Kiwi.new "Kenny "
bird6.print

File: unit11-hw/python/birdtest.py
# birdtest.rb | Tests out the menagerie of bird classes.
#
# Begun by: Dr. Jump for CS 315 at King's College.
# Completed by: Robert Lowman
# Date: 11/29/16
######################################################

import Duck
import Goose
import Owl
import Ostrich
import Kiwi
import Penguin
import Bird

bird0 = Bird.Bird('Hawkeye ')
bird0.printBird()

bird1 = Duck.Duck('Donald ')
bird1.printBird()

bird2 = Goose.Goose("Mother ")
bird2.printBird()

bird3 = Owl.Owl("Woodsey ")
bird3.printBird()

bird4 = Ostrich.Ostrich("Orville ")
bird4.printBird()

bird5 = Penguin.Penguin("Petey ")
bird5.printBird()

bird6 = Kiwi.Kiwi("Kenny ")
bird6.printBird()
